parser/ast/misc.go

```diff
@@ -1864,6 +1865,14 @@ type HandleRange struct {
 	End   int64
 }
 
+type StatementScope int
+
+const (
+	SessionScope StatementScope = iota
```

Comment:
It is better to add a `SessionNone` and use `0` as the default value. `session/instance/global` can be meaningless for the other `AdminStmt`.

Reply:
Update PTAL.